base-server/index.js

```diff
@@ -316,10 +317,12 @@ export class BaseServer {
     this.unbind = []
 
     this.connected = new Map()
+    this.queues = new Map()
     this.nodeIds = new Map()
     this.clientIds = new Map()
     this.userIds = new Map()
     this.types = {}
+    this.queueChannels = new Map().set('main', queueChannel)
```

Comment:
Will it be more readable to have flat `Map<queue + clientId, Queue>` here instead of more complex nested system?

Reply:
We can simplify it. Right now `BaseServer.queues` (https://github.com/muddv/logux-server/blob/queues/base-server/index.js#L320) is such a flat map, it  contains all current client's queues and  `queueChannels` is just needed to hold information about which actions can be processed in parallel.

Reply:
Actions who every client should be processed in parallel, so why do we need `queueChannels` with per-queue map?

Reply:
`queueChannels` are meant to be used by one client to process different actions in parallel, if the developer allows it during the creation of channel (https://github.com/muddv/logux-server/blob/queues/base-server/index.js#L509).
So for example, if server has 2 channels, like `users/` and `posts/`, Queue could be processing 2 actions from them, if needed.